### PR TITLE
Added legacy metadata editor to Pelagos for compability issues.

### PR DESCRIPTION
**Pull request by mickel1138**
Moved Legacy Metadata Editor to mail repo, to ensure compability with other components.

### CODE REVIEW

**Comment by mickel1138** [comment]
It's VERY important to use the correct config files.
One is needed in metadata-editor-start folder, and one in metadata-editor folder.
They are not the same file, but look similar.

**Comment by mickel1138** [comment]
Related to: PELAGOS-3818

File: src/Controller/Api/MetadataController.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            try {
+                $boundingBoxArray = $geoUtil->calculateGeographicBoundsFromGml($gml);
+            } catch (InvalidGmlException $e) {
+                $errors[] = $e->getMessage() . ' while attempting to calculate bonding box from gml';
+                $boundingBoxArray = array();
+            }
+        }
+
+        $generatedXmlMetadata = $metadataUtility->getXmlRepresentation($dataset, $boundingBoxArray);
+        $metadataFilename = preg_replace('/:/', '-', $dataset->getUdi()) . '-metadata.xml';
```
Comment (praneethpr) [inline src/Controller/Api/MetadataController.php]: We are not using the file name

Comment (mickel1138) [reply src/Controller/Api/MetadataController.php]: Yes, I took that out. Good find.

**Pull request merged**